SourceControl.md

```diff
@@ -0,0 +1,54 @@
+# General
+
+In the Triple .NET team we use Git for source control management. How you we [Git](https://git-scm.com) - whether it is via a GUI or the CLI - is up to us. Our commits however must stick to the guidelines:
+
+1. We must commit one change per commit
+2. We must adhere to the commit title and -description requirements
+3. We should not commit generated files
```

Comment:
In Umbraco generated files from database models should be commited as they are required. Generated migrations in Entity Framework should also be commited. Only like binaries or generated javascript files should not be commited, so this line should be changed to "Be careful when committing generated files" or something.

Reply:
That's exactly why i used **should**. On the other side of this coin, there's also possibilities for code generation on the build server which would then be recommended over checking-in generated code into source control.